14-location.Rmd

```diff
@@ -144,8 +140,8 @@ knitr::kable(tab,
 ## Create census rasters
  
 After the preprocessing, the data can be converted into a `SpatRaster` object (see Sections \@ref(raster-classes) and \@ref(raster-subsetting)) with the help of the `rast()` function.
-Setting its `type` argument to `xyz`, it requires an input data frame whose columns `x` and `y` represent coordinates on a regular grid.
-All the remaining columns (here: `pop`, `women`, `mean_age`, `hh_size`) will serve as input for the raster layers (Figure \@ref(fig:census-stack); see also `code/14-location-figures.R` in our github repository).
+When setting its `type` argument to `xyz`, an input data frame whose columns `x` and `y` will represent coordinates on a regular grid.
```

Comment:
Sentence is somehow wrong, I'll rewrite this.

Reply:
👍🏻